packages/react/src/components/field-label/field-label.stories.tsx

```diff
@@ -24,6 +25,32 @@ export const Playground: Story<FieldLabelProps> = (props: FieldLabelProps) => (
   <FieldLabel {...props} />
 );
 
+export const AsOptional: Story<FieldLabelProps> = ({
+  children,
+  ...restProps
+}: FieldLabelProps) => (
+  <FieldLabel {...restProps}>
+    <span>
+      {children}
+      &nbsp;
+      <span style={{ color: color('content-secondary') }}>(optional)</span>
```

Comment:
💅🏻
```suggestion
      <span style={{ color: color('content-secondary') }}> (optional)</span>
```

Reply:
I had like such originally, but then the code example becomes like such:

<img width="1015" alt="Screenshot 2021-02-10 at 15 34 43" src="https://user-images.githubusercontent.com/20243687/107532179-833d0900-6bb5-11eb-920e-30bd66f0d8f2.png">

So using `&nbsp;` might not look like best in code, but at least renders that code snippet better.

Although, now that I'm thinking probably it's not correct code example, so copy-pasting it would not reproduce the same output...

Reply:
The space will be missing, as React requires such either be set as `&nbsp;` or `{' '}`.

You just confused me now, shall I go with your suggestion to have a bit odd code samples but at least correct ones?

Reply:
I'm also confused, the example does include a `{' '}` 🤔 

I'm fine with either, folk accustomed to HTML will understand the `&nbsp;` easily, React-only people might have to think a bit.
Then `{' '}` is the opposite 🤷🏻